Documentation/install/guides/kops.rst

```diff
@@ -0,0 +1,35 @@
+**********************************
+Kubernetes Kops Installation Guide
+**********************************
+
+As of `kops<https://github.com/kubernetes/kops>`_ 1.9 release, Cilium is a supported CNI plugin for kops-deployed clusters.
+
+Cilium needs a newer kernel version then the default kops images provide (minimum kernel version for Cilium is 4.8), so you need to supply an `ami<https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html>`_ which will be new enough for Cilium to run on it.
+
+CoreOS images have new enough kernels for Cilium to run on them and are tested by kops developers, which makes them a perfect candidate.
+
+The latest stable CoreOS AMI can be found using aws cli:
+
+.. code:: bash
+
+    $ aws ec2 describe-images --region=us-east-1 --owner=595879546273 \
+        --filters "Name=virtualization-type,Values=hvm" "Name=name,Values=CoreOS-stable*" \
+        --query 'sort_by(Images,&CreationDate)[-1].{id:ImageLocation}'
+
+    {
+        "id": "595879546273/CoreOS-stable-1576.4.0-hvm"
+    }
+
+You also need to change the default etcd version used in kops, as Cilium needs at least version 3.1.0 (kops default is still on 2.0 branch).
```

Comment:
3.1.0 should be 3.1, right? we use 3.1.11 below.

Reply:
done